docs/config/README.md

```diff
@@ -807,12 +807,12 @@ it would have been `nixpkgs/pkgs`.
 | `truncation_symbol` | `""`                                               | The symbol to prefix to truncated paths. eg: "…/"                                |
 | `repo_root_style`   | `None`                                             | The style for the root of the git repo when `truncate_to_repo` option is set to false.|
 | `home_symbol`       | `"~"`                                              | The symbol indicating home directory.                                           |
+| `convert_slash`     | `true`                                             | Convert slash on output on Windows                                               |
```

Comment:
Can we get a more detailed description here? Just looking at this, I have no idea what it means to convert slash output.

What's the intention behind this option? My impression is that, when enabled, it will display paths on windows using backslashes instead of forward slashes? In that case, I think it might be better to name it for what it does (e.g. `win_display_backslash`, though I'd like a shorter name) instead of how it accomplishes this.